Fee Market Meetings/Meeting 08.md

```diff
@@ -0,0 +1,293 @@
+like take into account like the higher one and the lower one or something and then you just update that maybe even just once every. So many blocks and as long as you can do that efficiently and as long as basically that is good enough that like on the high end your sorting is
+never is still absolutely like perfectly precise. But again I am right now not confident enough that this would work already. But that basically would be like currently, like one of my candidates for like a very simple heuristic that might be efficient but still good enough, yeah if that makes sense thanks. 
+
+**JosephC**: Yeah that does make sense.
+
+**Tim Beiko**: Yeah, cool anything else on the transaction pool management. 
+
+## 1.2 Update on the large state testnet
+
+**Tim Beiko**: Okay next up Abdel do you want to give a quick update on the large state testnet generator and where we are at?
+
+**Abdulhamid Bakhta**: Okay, yes so we currently have set up the new testnet so, this will be a
+profound testnet and the goal is to have a state comparable to mainnet. So far we have generated 100 million accounts and we are now using a smart contract and we aim to generate 100 million entries in this smart contract. And yeah when this will be ready we will share the url of the different nodes and the block explorer and the headset so that other clients can think of this new testnet and yeah that is pretty much. So, we have four nodes running and the generator is still running and I will share everything when it will be ready. That's it.
+
+**Tim Beiko**: Cool yeah and I think for that once we have it up and once we have you know the testnet up and running on Besu and Geth then we get Nethermind and get syncing to it. I think we should probably just **schedule a time to then spam it with a kind of transaction and gather metrics from all three clients**. Hopefully we can gather metrics and it does not you know just fall over but if it falls we fix it and try again. But I think, if we have at least one or a few shots of like saying look we spammed the testnet for you know two hours with transactions and like the node stood up. I think that is like more than the worst case. We would see on mainnet because in two
+hours the base you would probably go up. Like you know a hundred thousand x or a million x and it is just not realistic to even do such an attack.
+
+**Ramil Amerzyanov**: Hi it is Ramil, I just do not join it. Sorry for being late.
+
+**Tim Beiko**: No worries. Any thoughts or comments on the testnet.
+
+**Nethermid**: I think it would be important to make sure that every single node is publishing transactions of their own. Respective transaction pools so we know that we can not only consume the transaction load but also that every single one can generate them.
```

Comment:
```suggestion
**Nethermind**: I think it would be important to make sure that every single node is publishing transactions of their own. Respective transaction pools so we know that we can not only consume the transaction load but also that every single one can generate them.
```

Can you please update throuhgout? Thanks!

Reply:
Thanks for review . 
I have updated the notes with your suggestions.